src/js/renderShabad.js

```diff
@@ -29,6 +50,18 @@ function metaData(data) {
   gurmukhi_meta.push('<a href="/ang?ang=' + data.source.pageno + '&amp;source=' + data.source.id + '">' + page_type_gurmukhi + ' ' + data.source.pageno + '</a>');
   english_meta.push('<a href="/ang?ang=' + data.source.pageno + '&amp;source=' + data.source.id + '">' + page_type_english + ' ' + data.source.pageno + '</a>');0
 
+  if(typeof nav != "undefined") {
+    let link = navLink(nav,data.source.id);
+
+    if (typeof nav.previous != "undefined") {
+      $meta.appendChild(h('div', { class: 'shabad-nav left'}, '<a href="' + link + nav.previous + '">&lt;</a>'));
+    }
+
+    if (typeof nav.next != "undefined") {
+      $meta.appendChild(h('div', { class: 'shabad-nav right'}, '<a href="' + link + nav.next + '">&gt;'));
+    }
```

Comment:
Can we do this instead? Also you forgot to close the anchor tag for right button. This would allow us to convert it into JSX in future, making it also possible to port to react easily.

```diff
- $meta.appendChild(h('div', { class: 'shabad-nav left'}, '<a href="' + link + nav.previous + '">&lt;</a>'));
+ $meta.appendChild(h('div', { class: 'shabad-nav left'}, h('a', { href: link + nav.previous }, '&lt;')));
...
- $meta.appendChild(h('div', { class: 'shabad-nav right'}, '<a href="' + link + nav.next + '">&gt;'));
+ $meta.appendChild(h('div', { class: 'shabad-nav right'}, h('a', { href: link + nav.next }, '&gt;'));
```